datafusion/physical-expr-common/src/utils.rs

```diff
@@ -134,6 +135,33 @@ pub fn limited_convert_logical_expr_to_physical_expr(
     }
 }
 
+pub fn limited_convert_logical_expr_to_physical_expr_with_dfschema(
+    expr: &Expr,
+    dfschema: &DFSchema,
+) -> Result<Arc<dyn PhysicalExpr>> {
+    match expr {
+        Expr::Alias(Alias { expr, .. }) => Ok(
+            limited_convert_logical_expr_to_physical_expr_with_dfschema(expr, dfschema)?,
+        ),
+        Expr::Column(col) => {
+            let idx = dfschema.index_of_column(col)?;
```

Comment:
I found that I need dfschema in order to get the correct field, so I change all the UDAF to `create_aggregate_expr_with_dfschema` which is used internally for UDAF only.

Reply:
After removing `limited_convert_logical_expr_to_physical_expr_with_dfschema`, I think we don't need `DFSchema` anymore

Reply:
So does that mean `limited_convert_logical_expr_to_physical_expr_with_dfschema` will be temporary?